src/calliope/math/spores.yaml

```diff
@@ -2,9 +2,37 @@ import:
   - plan.yaml
 
 constraints:
-  cost_sum_max:
+  total_system_cost_max:
+    description: >
+      Limit total system cost. Conceived for use in SPORES mode to apply a maximum
+      relaxation to the system cost compared to the least-cost feasible option.
+    foreach: [costs]
```

Comment:
Is it normal for this to iterate over all types of costs? In the docs, we say the default is 'monetary', but this doesn't seem to reflect it. Or is it intended that it is open to all costs here, but 'by default' users tend to then specify only one type of cost in their use of this?

Reply:
True, currently it would apply a constraint limiting all cost classes to `least_cost_val * (1 + spores_slack)`  (note now that the SPORES score _isn't_ a cost). I suppose this would matter if we were tracking other costs in the model (e.g. CO2 emissions). We could limit this to `monetary` in the math.